control-plane/pkg/reconciler/broker/namespaced_broker.go

```diff
@@ -148,7 +139,7 @@ func (r *NamespacedReconciler) createReconcilerForBrokerInstance(broker *eventin
 
 // getManifest returns the manifest that is transformed from the BaseDataPlaneManifest with the changes
 // for the given broker
-func (r *NamespacedReconciler) getManifest(broker *eventing.Broker) (mf.Manifest, error) {
+func (r *NamespacedReconciler) getManifest(ctx context.Context, broker *eventing.Broker) (mf.Manifest, error) {
```

Comment:
should we name this to something more like `reconcileDataPlaneManifest`, than a meaningless `get`?

Ok to do that in a follow up PR, of course